components/omnibox/browser/BUILD.gn

```diff
@@ -2,31 +2,16 @@ source_set("browser") {
   # Only //components/omnibox/browser target can depend on this target
   # because this target expands //components/omnibox/browser implementation.
   visibility = [ "//components/omnibox/browser" ]
-  # Remove when https://github.com/brave/brave-browser/issues/10624 is resolved
-  check_includes = false
-
   sources = [
-    "brave_omnibox_client.cc",
-    "brave_omnibox_client.h",
     "constants.cc",
```

Comment:
why did you leave these?

Reply:
I only moved files which have the dependency of //chrome/browser. Should I move them as well?